lab-davidw/test/test-readFiles.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const expect = require('chai').expect;
+const eatFiles = require('../lib/readFiles.js');
+
+
+describe('readFile()', function() {
+  let mockDataPaths = [`${__dirname}/../data/one.txt`, `${__dirname}/../data/two.txt`, `${__dirname}/../data/three.txt`];
+  it('should return hex value for one.txt', done => {
+    var mockOutput = ['4865726527732061', '57686f2077617320', '54696c6c20746865'];
```

Comment:
So you dont need to create this array in every it block. You can actually instansiate this right after line 9, then refer it in all of your tests.